samples/pom.xml

```diff
@@ -16,9 +16,9 @@
     Removing or replacing it should not affect the execution of the samples in anyway.
   -->
   <parent>
-    <groupId>com.google.cloud.samples</groupId>
-    <artifactId>shared-configuration</artifactId>
-    <version>1.2.0</version>
+    <artifactId>google-cloud-retail-parent</artifactId>
```

Comment:
Why are you overriding shared-configuration? This should be the parent for all samples

Reply:
I have tried to set up configuration because IDE (IntelliJ IDEA) is not recognized Maven modules properly. I can't run my code if Maven modules relations are not configured.

Reply:
reverted